modules/home/src/lib/latest-news/latest-news.component.html

```diff
@@ -0,0 +1,50 @@
+<body>
+  <h2 class="p-10px">Últimas Notícias</h2>
+  <section class="container p-10px">
+    <div class="main">
+      <img
+        src="main-notice.svg"
+        alt="imagem noticia principal"
+        class="main-image"
+      />
+      <h3>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua
+      </h3>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Sit amet purus
+        gravida quis blandit. Id volutpat lacus laoreet non curabitur. Nunc
+        scelerisque viverra mauris in aliquam sem fringilla. Cursus vitae congue
+        mauris rhoncus aenean. Ornare lectus sit amet est placerat. Tortor at
+        risus viverra adipiscing at in tellus integer.
+      </p>
+    </div>
+    <div class="side">
+      <div class="side-item mb-10px">
+        <img
+          src="side-notice.svg"
+          alt="imagem noticia lateral"
```

Comment:
```suggestion
          alt="Notícia lateral"
```

Trocar o `alt` para que o Sonar não relate bug de acessibilidade